package.json

```diff
@@ -15,6 +15,7 @@
   },
   "scripts": {
     "cp": "mkdir -p ./dev/public/shiki && cp -r ./node_modules/shiki/{dist,languages,samples,themes} ./dev/public/shiki",
+    "cp:shikifix": "cp node_modules/.pnpm/vscode-oniguruma@1.7.0/node_modules/vscode-oniguruma/release/onig.wasm ./dev/public/shiki/dist/",
```

Comment:
Could you help me understand what this does?

Reply:
The code examples rely on Shiki, but Shiki currently has a quirk when used clientside and that script fixes it.